inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -468,4 +470,38 @@ public function update_safelist_items( $old_version ) {
 
 		update_option( 'wp_rocket_settings', $options );
 	}
+
+	/**
+	 * Display a notification on wrong license.
+	 *
+	 * @return void
+	 */
+	public function display_wrong_license_notice() {
+		if ( ! $this->can_display_notice( false ) ) {
```

Comment:
I don't think the parameter is necessary, we don't want to display the notice if the option is not enabled

Reply:
@Tabrisrp we want as we don't activate the option if the client has not license